### PR TITLE
yosys version update

### LINKED ISSUE #1507: OpenROAD `./etc/DependencyInstaller.sh` installs `yosys-0.33` while flow uses different version
### Describe the bug

OpenROAD's `./etc/DependencyInstaller.sh` installs `yosys-0.33`;

https://github.com/The-OpenROAD-Project/OpenROAD/blob/347459a5c91f473de611c0c0f403eb4880e8d339/etc/DependencyInstaller.sh#L12-L13

While the OpenROAD flow script seems to use a version [bc027b2](https://github.com/The-OpenROAD-Project/yosys/commit/bc027b2cae9a85b887684930705762fac720b529) [on Jan 19, 2022](https://github.com/The-OpenROAD-Project/yosys/commit/bc027b2cae9a85b887684930705762fac720b529) see;

https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/tree/master/tools

### Expected Behavior

`OpenROAD` and `OpenROAD-flow-scripts` use the same version of Yosys.

### Environment

```shell
N/A
```

### To Reproduce

N/A

### Relevant log output

_No response_

### Screenshots

_No response_

### Additional Context

_No response_

### PR DESCRIPTION
**Pull request by vvbandeira**

### CODE REVIEW

**Comment by luarss** [comment]
@vvbandeira Can you remind me to update the documentation when this is merged. Thanks!

**Comment by mithro** [comment]
This change needs to be updated?

It should be also updating Yosys to 2584903a0605cc7ffdfc1996254ccc1f548403f2 (`yosys-0.33`) to match OpenROAD version?

File: build_openroad.sh [hunk 1]
```diff
@@ -20,10 +20,7 @@ OPENROAD_APP_BRANCH="master"
 INSTALL_PATH="$(pwd)/tools/install"
 
 YOSYS_USER_ARGS=""
-YOSYS_ARGS="\
-CONFIG=gcc \
```
Comment (mithro) [inline build_openroad.sh:24]: Removing the `CONFIG=gcc` seems to be causing some issue with clag on the CI.
`#8 0.412 /bin/sh: clang: command not found`
https://jenkins.openroad.tools/blue/organizations/jenkins/OpenROAD-flow-scripts-Public%2Fpublic_tests_all-pr/detail/PR-1468-head/6/pipeline/595/#step-600-log-2164

**Comment by vvbandeira** [comment]
replaced by #1575

**Pull request closed without merging**